en_ewt-ud-train.conllu

```diff
@@ -197015,7 +197015,7 @@
 # sent_id = reviews-348369-0006
 # text = Tmobile want to Send of my phone and i didn't want to go thru that...
 1	Tmobile	Tmobile	PROPN	NNP	Number=Sing	2	nsubj	2:nsubj|4:nsubj:xsubj	_
-2	want	want	VERB	VBZ	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
+2	want	want	VERB	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
```

Comment:
Isn't this the British collective plural? VBP?

Reply:
Fixed, thanks.